recipe page/app.css

```diff
@@ -113,7 +112,84 @@ body {
     color: red;
 }
 
-/* Why doesn't this work? */
-/* input:checked {
+.buttonsDiv {
+    padding-top: 10px;
+    padding-bottom: 20px;
+}
+
+.buttons {
+    display: inline-block;
+    padding: 15px 25px;
+    font-size: 24px;
+    cursor: pointer;
+    text-align: center;
+    text-decoration: none;
+    outline: none;
+    color: #fff;
+    background-color: #aad7ee;
+    border: none;
+    border-radius: 30px;
+    box-shadow: 0 9px #999;
+}
+
+.buttons:hover {
+    background-color: #71c0e8
+}
+
+.buttons:active {
+    background-color: #3bace5;
+    box-shadow: 0 5px #666;
+    transform: translateY(4px);
+}
+
+#footerContainer1 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: x-large;
+}
+
+#footerContainer2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: large;
+}
```

Comment:
one thing i might recommend is giving these slightly more descriptive names --  instead of `1` vs `2` ,what do each of the containers do?